include/fastdds/dds/domain/qos/DomainParticipantQos.hpp

```diff
@@ -31,13 +31,15 @@ namespace dds {
 /**
  * Class DomainParticipantQos, contains all the possible Qos that can be set for a determined participant.
  * Please consult each of them to check for implementation details and default values.
+ *
  * @ingroup FASTDDS_QOS_MODULE
  */
 class DomainParticipantQos
 {
 public:
 
     //! User defined flow controllers to use alongside.
+    //
     //! @since 2.4.0
```

Comment:
Don't you have to use the `/*` instead of `//!`. In `include/fastdds/dds/core/policy/QosPolicies.hpp` you have replaced on method for the other.

Reply:
Done